.env.local

```diff
@@ -10,3 +10,6 @@ HYDRA_HOST_STAGING=https://hydra.${NEXT_PUBLIC_SERLO_DOMAIN_STAGING}
 HYDRA_CLIENT_ID_LOCAL=frontend.serlo.org
 HYDRA_CLIENT_SECRET_LOCAL=frontend.serlo.org
 HYDRA_HOST_LOCAL=http://localhost:4444
+KRATOS_HOST_STAGING=https://kratos.${NEXT_PUBLIC_SERLO_DOMAIN_STAGING}
+KRATOS_HOST_LOCAL=http://localhost:4433
+ORY_SDK_URL="(NEXT_PUBLIC_ENV=local)?KRATOS_HOST_LOCAL:KRATOS_HOST_STAGING"
```

Comment:
ORY_SDK_URL is maybe unnecessary, check it

Reply:
Seems to be unused. will remove it now, let's see if it breaks something :)

Reply:
Just for info: Its use is hidden in https://github.com/serlo/frontend/blob/ded405a75d9d924a0fc1880becbb46493bddfbd9/src/pages/api/.ory/%5B...paths%5D.ts
See https://github.com/ory/integrations#nextjs
But if you go to the [source code](https://github.com/ory/integrations/blob/main/src/next-edge/index.ts#L44-L58) you see that we already overwrite it